ajax/libs/leaflet-hash/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": "Michael Lawrence Evans",
+  "name": "leaflet-hash",
+  "description": "linkable location hashes for leaflet",
+  "version": "0.2.1",
+  "homepage": "https://github.com/mlevans/leaflet-hash",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mlevans/leaflet-hash.git"
+  },
+  "main": "leaflet-hash.js",
+  "optionalDependencies": {},
+  "engines": {
+    "node": "*"
+  },
+  "npmName": "leaflet-hash",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "*.js"
```

Comment:
Please use `leaflet-hash*.js` because root is not a dist folder. Should limit the wildcard.